libs/common-hooks/src/useAvailableChains.ts

```diff
@@ -13,13 +11,14 @@ import { useFeatureFlags } from './useFeatureFlags'
  * Set the flag in this hook.
  */
 export function useAvailableChains(): SupportedChainId[] {
-  // 1. Load feature flag for chain being enabled
-  const { isBaseEnabled } = useFeatureFlags()
+  // // 1. Load feature flag for chain being enabled
+  // const { isBaseEnabled } = useFeatureFlags()
+
+  // return useMemo(
+  //   // 2. Conditionally build a list of chain ids to exclude
+  //   () => getAvailableChains(isBaseEnabled ? undefined : [SupportedChainId.BASE]),
+  //   [isBaseEnabled],
+  // )
```

Comment:
Why again? https://github.com/cowprotocol/cowswap/pull/5234

Reply:
I see..., I did it in `main` branch. This is not in prod?